### PR TITLE
Fix GraphNet and kerasNet submodules bug

### PR DESCRIPTION
**Pull request by yangw1234**
Fix GraphNet and kerasNet submodules bug

### CODE REVIEW

File: pyzoo/zoo/__init__.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 prepare_env()
 if not SparkFiles._is_running_on_worker:
     check_version()
+JavaCreator.add_creator_class("com.intel.analytics.zoo.pipeline.api.keras.python.PythonZooKeras")
```
Comment (zhichao-li) [inline pyzoo/zoo/__init__.py]: remove this?

**Comment by yangw1234** [comment]
pending on #132

File: pyzoo/test/zoo/pipeline/api/keras/test_simple_integration.py [hunk 2]
```diff
@@ -124,6 +125,22 @@ def test_sequential_to_model(self):
...
+        model = Model([x1, x2], [y1, y2])
+        assert len(model.layers) == 4
+
+    def test_keras_net_flatten_layers(self):
+        x1 = Input(shape=(8, ))
+        x2 = Input(shape=(6, ))
+        y1 = Dense(10)(x1)
+        y2 = Dense(10)(x2)
+        model = Model([x1, x2], [y1, y2])
+        assert len(model.flattened_layers()) == 4
```
Comment (hkvision) [inline pyzoo/test/zoo/pipeline/api/keras/test_simple_integration.py]: Can these two unit tests be combined into one? As the model are all the same... just use two assert

**Comment by jason-dai** [comment]
Is this ready for merge?

**Comment by yangw1234** [comment]
pr validation pass

**Pull request merged**